File: python/bc1865.py
count = int(input())
res = ''
for i in range(0, count):
    cVal = input().split(" ")

    name = cVal[0]
    force = int(cVal[1])

    if name == "Thor": res = 'Y'
    else: res = 'N'

    print(res)

File: cpp/bc1865.cpp
#include <iostream>

using namespace std;

int main()
{
	int count, force;
	string name;
	
	cin >> count;
	
	for(int i = 0; i < count; i++)
	{
		char res;
		
		cin >> name >> force;
		
		if(name == "Thor") res = 'Y';
		else res = 'N';
		
		cout << res << endl;
	}
}
